validate.lic

```diff
@@ -478,6 +478,12 @@ class DRYamlValidator
       .each { |skill| error("Magic skill in crossing_training does not have a training_nonspells defined: #{skill}") }
   end
 
+  def assert_that_check_discern_timer_is_deprecated(settings)
+    return unless settings.check_discern_timer
+
+    warn('The check_discern_timer setting is deprecated. It is now check_discern_timer_in_hours for clarity. Please adjust accordingly.')
```

Comment:
💡 I would expect this to be an `error` given the old setting has been removed entirely.

Reply:
Aaah, was changing it, lol.